src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/BindNeverAttribute.cs

```diff
@@ -7,8 +7,8 @@ namespace Microsoft.AspNetCore.Mvc.ModelBinding
 {
     /// <summary>
     /// Indicates that a property should be excluded from model binding. When applied to a property, the model binding
-    /// system excludes that property. When applied to a type, the model binding system excludes all properties of that
-    /// type.
+    /// system excludes that property. When applied to a type, the model binding system excludes all properties within
+    /// that type.
```

Comment:
Could but I'm not sure it's much more clear than the `with` / `within` a type distinction I've made. I'll let you make the call&hellip;